### PR TITLE
Linphone core update.

### PR DESCRIPTION
**Pull request by shareefalis**
See android pr for information about it

### CODE REVIEW

File: submodules/linphone [hunk 1]
```diff
@@ -1 +1 @@
-Subproject commit 82e395bc9b140f7f97b7ab9b41d8e5aa41f70965
+Subproject commit 0e996357d70a606a86d97af7d28eaa8451f09952
```
Comment (ianblenke) [inline submodules/linphone:1]: The diff between these two shows some pretty substantial vCard changes.

I've built it locally, and it does build and run, but we really need to stop following upstream so closely if we hope to release GA soon.

This means: no more upstream merges for now, unless we run into some huge problems that require it. Ok?

Comment (shareefalis) [inline submodules/linphone:1]: The issue is that  h263  isn't enabled in the current build and that need to be fixed..
See commit for the fix.
https://github.com/BelledonneCommunications/mediastreamer2/commit/863a2d612832cf8722ce345246156baba9cc93be

**Comment by zackm0571** [comment]
+1 for H.263 fixes

**Comment by zackm0571** [comment]
If this causes any major issues we will revert

**Pull request merged**